pom.xml

```diff
@@ -440,6 +504,20 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.openmrs.module</groupId>
+            <artifactId>uilibrary-api</artifactId>
+            <version>${uilibraryVersion}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
```

Comment:
No, this cannot go here.  This must be moved into the omod project, unless we use a dependencyManagement section (which we don't have).  The api cannot contain omod artifacts.

Reply:
Fair enough... will do a little testing to see if this does what is expected when it's in the omod, but otherwise we can just punt on the REST updates for now.

Reply:
Looks like it doesn't pick up the REST web services version upgrade if it's not referenced in that pom... will just not worry about that for now, we can upgrade REST web services manually as need be for now.